test/navigation/NavigatorForCorrectReturnSpec.scala

```diff
@@ -255,6 +255,20 @@ class NavigatorForCorrectReturnSpec extends SpecBase with DataHelper {
 
   }
 
+  "Remove Secondary Warehouse Details Page" - {
+    List(NormalMode, CheckMode).foreach(mode => {
+      s"must go from $RemoveWarehouseDetailsPage to $SecondaryWarehouseDetailsPage in $mode when user selects yes" in {
+        navigator.nextPage(RemoveWarehouseDetailsPage, mode, UserAnswers("id", SelectChange.CorrectReturn, contactAddress =
+          contactAddress).set(RemoveWarehouseDetailsPage, true).success.value) mustBe controllers.correctReturn.routes.SecondaryWarehouseDetailsController.onPageLoad(mode)
```

Comment:
Can just be `routes.SecondaryWarehouseDetailsController.onPageLoad(mode)` as already imported, same for below